docs/guide/faq.rst

```diff
@@ -158,3 +158,41 @@ This does.
 
 TL;DR: Make sure you're importing the appropriate implicits. They define
 methods that extend GeoTrellis types.
+
+How do I resolve a Guava / other library dependency incompatibility issue(s)?
+===================================
+
+Full possible exception message:
+
+.. code::
+
+    Caused by: java.lang.IllegalStateException: Detected Guava issue #1635 which indicates that a version of Guava less
+    than 16.01 is in use.  This introduces codec resolution issues and potentially other incompatibility issues in the driver.
+    Please upgrade to Guava 16.01 or later.
+
+
+GeoTrellis depends on a huge number of complex dependencies that may cause a dependency hell. One of such dependency
+is a guava library. ``GeoTrellis ETL`` and ``GeoTrellis Cassandra`` depend on ``Guava 16.01``, but Hadoop depends on ``Guava 11.0.2``
+that causes strong libs incompatibility runtime issues. When two different versions of the same library both available in a
+Spark classpath and in a fat assembly jar, Spark would use library version from its classpath.
```

Comment:
`in a Spark classpath` -> `in the Spark classpath`

`Spark would use` -> `Spark will use`